src/model/otr_model.rs

```diff
@@ -697,7 +668,15 @@ mod tests {
         }
     }
 
-    /// This test ensures the rating changes are exactly as described in our sample match.
+    /// This test ensures the rating changes are exactly as described in our
+    /// sample match. The following values for parameters are assumed (and
+    /// this test should be updated if any of them are changed):
+    ///
+    /// A gamma_override function of 1/k
+    /// BETA = 150
+    /// WEIGHT_A = 0.9 (and therefore WEIGHT_B = 0.1)
+    /// GAME_CORRECTION_EXPONENT = 0.5
+    /// GAME_CORRECTION_BASE = 8
```

Comment:
```suggestion
    /// GAME_CORRECTION_BASE = 8.0
```
I assume thats a float?